examples/airflow-torch-training/callables.py

```diff
@@ -1,54 +1,59 @@
-import runhouse as rh 
 import logging
 
-from torch_example_for_airflow import SimpleTrainer, DownloadData
+import runhouse as rh
+
+from torch_example_for_airflow import DownloadData, SimpleTrainer
 
 # Configure logging
 logger = logging.getLogger(__name__)
 
 
 def bring_up_cluster_callable(**kwargs):
     logger.info("Connecting to remote cluster")
-    cluster = rh.ondemand_cluster(name="a10g-cluster", instance_type="A10G:1", provider="aws").up_if_not()
-    #cluster.save() ## Use if you have a Runhouse Den account to save and monitor the resource. 
+    rh.ondemand_cluster(
```

Comment:
could also add "F841" to the pyroject.toml `[tool.ruff.lint.per-file-ignores]` examples section, but this works too